src/app/login/login.controller.js

```diff
@@ -101,9 +112,22 @@
                 password: $scope.password
             };
         }
+
+        function getRememberedLoginInfo() {
+            if ($cookies.get('user_credentials') != null && $cookies.get('user_credentials') != undefined) {
+                var credentials = $base64.decode($cookies.get('user_credentials')).split(":");
+                return {
+                    email: credentials[0],
+                    password: credentials[1]
+                }
+            } else {
+                return false;
+            }
+        }
+
     }
 
-    LoginController.$inject = ['$scope', '$cookies', '$base64', '$httpParamSerializer', 'LoginService'];
+    LoginController.$inject = ['$scope', '$cookies', '$base64', '$httpParamSerializer', 'LoginService', '$location'];
 
     angular
```

Comment:
Single responsibility principle - one component per file. I'd have no problem if the controller was declared via function, but here you are declaring a new component.